app/packages/core/src/components/Filters/categoricalFilter/ResultComponent.tsx

```diff
@@ -1,7 +1,10 @@
+import * as fos from "@fiftyone/state";
 import React from "react";
+import { useRecoilValue } from "recoil";
 import { V } from "./CategoricalFilter";
 
 const ResultComponent = ({ value: { value, count } }: { value: V }) => {
+  const isFilterMode = useRecoilValue(fos.isSidebarFilterMode);
```

Comment:
I would propose making `V["count"]` an optional type to avoid using this hook here. Just a suggestion